docs/organizer-guide/confs/community-events.rst

```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
+They are **not** officially organized by Write the Docs organizers,
+but they are sister events that share a similar goal.
```

Comment:
sister > sibling? (not even sure that the family association fits here, maybe "adjacent" or "friendly" or something else)

Reply:
Yeah, not entirely sure about the siblingishness. @thatdocslady makes good points.

Could just for for "related events with similar goals and values" or some such nonsense.